airbyte-server/src/main/java/io/airbyte/server/handlers/WorkspacesHandler.java

```diff
@@ -188,9 +188,10 @@ public WorkspaceRead updateWorkspace(final WorkspaceUpdate workspaceUpdate) thro
   public NotificationRead tryNotification(final Notification notification) {
     try {
       final NotificationClient notificationClient = NotificationClient.createNotificationClient(NotificationConverter.toConfig(notification));
-      final String message = String.format("Hello World! This is a test from Airbyte to try %s notification settings",
-          notification.getNotificationType());
-      if (notificationClient.notify(message)) {
+      final String messageFormat = "Hello World! This is a test from Airbyte to try %s notification settings for sync %s";
+      final boolean failureNotified = notificationClient.notifyFailure(String.format(messageFormat, notification.getNotificationType(), "failures"));
+      final boolean successNotified = notificationClient.notifySuccess(String.format(messageFormat, notification.getNotificationType(), "successes"));
+      if (failureNotified || successNotified) {
```

Comment:
Do we really need to send both success and failure (two notifications) as tests to try if the webhook works?

Reply:
I do not have strong preference. Only reasoning I had was the flags control success and/or failure notifications so it seemed right to send both if both are enabled and send one or other if only one of them (or none if none of them) is enabled.

If you feel strongly, I can drop one. Let me know.